src/formio/components/checkbox.ts

```diff
@@ -0,0 +1,17 @@
+import {InputComponentSchema} from '..';
+
+type Validator = 'required';
+type TranslatableKeys = 'label' | 'description' | 'tooltip';
+
+export type CheckboxInputSchema = InputComponentSchema<boolean, Validator, TranslatableKeys>;
+
+/**
+ * @group Form.io components
+ * @category Concrete types
+ */
+export interface CheckboxComponentSchema
+  extends Omit<CheckboxInputSchema, 'hideLabel' | 'disabled'> {
+  type: 'checkbox';
+  defaultValue?: boolean;
```

Comment:
looking at the generated JSON by the formio builder, `defaultValue` is initialized to `false`, so I think we can drop the optional behaviour.

```suggestion
  defaultValue: boolean;
```